api/db/migrations/20220429143417_add-scoring-columns-to-table-complementary-certifications.js

```diff
@@ -0,0 +1,20 @@
+exports.up = async function (knex) {
+  await knex.schema.table('complementary-certifications', (table) => {
+    table.float('minimumReproducibilityRate').nullable();
```

Comment:
Je ne suis personnellement pas fan de l'usage du type float, qui est par nature imprécis.
Je lui préfère le type decimal (aussi appelé numeric je crois).

Sur le type float :

> Because of these design choices, floating point numbers can work with numbers with large number of decimals efficiently, but not always exactly. The internal representation of numbers may cause slight differences between the input and output. This can cause unexpected behavior when comparing values, doing floating point math, or performing operations that require exact values

Reply:
On va mettre un NUMERIC(5, 2) (decimal côté knex) qui est la valeur du `reproductibilityRate` côté `assessment-results`.